Include/internal/pycore_gc.h

```diff
@@ -68,6 +69,19 @@ static inline int _PyObject_GC_MAY_BE_TRACKED(PyObject *obj) {
     return 1;
 }
 
+#ifdef Py_GIL_DISABLED
+
+static inline int _PyObject_GC_IS_SHARED(PyObject *op) {
```

Comment:
I think it's worth adding a comment describing what `_PyGC_BITS_SHARED` is about.